Doprez.Stride/Extensions/EntityComponentExtensions.cs

```diff
@@ -13,30 +13,56 @@ public static class EntityComponentExtensions
     /// <typeparam name="T"></typeparam>
     /// <param name="entity"></param>
     /// <returns></returns>
-    public static T GetComponent<T>(this Entity entity)
+    public static T GetComponent<T>(this Entity entity) where T : EntityComponent
```

Comment:
The change for Linq is completely understandable. Although I dont want to add the requirement of EntityComponent here. The reason I made this method was due to the limitation of not getting Interface types from Entities compared to the existing Get<T>() from Strides default method.

Reply:
I removed the generic limits